docs/docs/overview/quickstart/get-started-with-nextjs.md

```diff
@@ -0,0 +1,319 @@
+
+This is a simple function that calls the **`create`** method on the **`topics`** property of the **`novu`** instance. This method creates a new topic in the Novu system using the provided **`key`** and **`name`** values.
+
+If you test this on your local machine, you should get something like this:
+
+![testing create method](https://res.cloudinary.com/dxc6bnman/image/upload/v1686780448/guides/Untitled_1_i0yw1r.png)
+
+> Note how the return object contains the key I’d sent from my request body. That signals successful creation!
+
+## Adding subscriber to a Topic
+
+The code for adding a subscriber to a previously created topic is as follows:
+
+```jsx
+import { Novu } from '@novu/node';
+import CircularJSON from 'circular-json';
+
+export default async function addSub(req, res) {
+    try {
+        const novu = new Novu(<YOUR_NOVU_API_KEY>);
+        if (req.method === 'POST') {
+            // Get the subscriber ID from the request body
+            const subscriberId = req.body.subscriberId;
+            // Get the topic key from the request body
+            const topicKey = req.body.topicKey;
+            // Call Novu SDK to add the subscriber to the topic
+            const result = await novu.topics.addSubscribers(topicKey, {
+                subscribers: [subscriberId],
+            });
+            // Return the result as JSON response
+            res.status(200).json(CircularJSON.stringify(result));
+        }
+    } catch (error) {
+        console.log(error);
+        res.status(500).json(CircularJSON.stringify({ message: error.message }));
+    }
+}
+```
+
+If you see the code above closely, you’ll see that we first establish a connection to Novu using the Novu API key. Then we extract `subscriberID` and topicKey from the request body and call the **`addSubscribers`** method on the **`topics`** property of the **`novu`** instance, passing the topic key and an object with an array of subscribers.
+
+This adds the subscriber with the `subscriberID` we’d passed to the array of subscribers, which in the above case contains just one subscriber.
+
+If you check this on Postman, the array returned will contain the `subsriberID` (highlighted in the result) we’d passed in the request body signaling that it was added, as shown below:
+
+![The returned array contains 'subscriberID'](https://res.cloudinary.com/dxc6bnman/image/upload/v1686780607/guides/Untitled_2_za1uqr.png)
+
+After creating a topic and adding subscribers to the topic, we’ll now proceed to send a notification to all subscribers (we only have one subscriber in our topic) of a topic. This is what Topics are used for - sending bulk notifications.
+
+## Sending notifications to a topic
+
+Sending notifications to a topic is pretty straightforward. You just need to extract the topic key to which you want to send notifications and trigger Novu’s method on that topic key with the message in the payload:
```

Comment:
We need to be careful to avoid using sentences like "pretty straightforward" because it might not be for many people.  We can reword your sentence from "Sending notifications to a topic is pretty straightforward" to "Sending notifications to a topic is not a complex task".

Reply:
You just need to extract => You need to extract..

Reply:
Okay, will take this into account from now on. Thanks!